arch/arm/boot/dts/overlays/mira220.dtsi

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: GPL-2.0-only
+// Definitions for MIRA220 camera module on VC I2C bus
+
+
+//#include <dt-bindings/gpio/gpio.h>
```

Comment:
Is this commented out section left here for a reason? If so, explain why in a comment, otherwise delete.